<html>
<head>
<script src="resources/profiler-test-JS-resources.js"></script>
<script>
console.profile("Throw within an eval.");
function testStart()
{
    insertNewText();
    endTest();
}
</script>
<script> eval("throw('I threw up');");</script>
</head>

<body onload="testStart()">
This page throws an exception from within eval().
<br>
<br>
To use this test, load it in the browser then load the WebInspector and look at
the profile.  The profiler should not crash and still show the stack under
insertnewText().
<div id="output"></div>
</body>
</html>
